docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - '80:80'
 
   frontend:
-    image: danlynn/ember-cli:3.21.0-node_12.18
+    image: danlynn/ember-cli:4.8.0
```

Comment:
Should be `3.28.6` like in the `package.json`

Reply:
Yes it *should* be, but... with the forced update (older package versions not available anymore) to `adfinis/eslint-config: ^2.0.0` we are required to satisfy `node >= 18`. The `3.28.x` tags are all on `14` or `16`. Another option would be to use the `--ignore-engines` flag on the `yarn install` command. What would you prefer?

Reply:
Ah shite I see.. Let's keep it that way as it will still use the version specified in the dependencies for all commands :+1: